src/index.js

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from 'mongodb';
+
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+async function getDB() {
+  try {
+    await client.connect();
+    return client.db('kaende');
+  } catch(ex) {
+    console.error(ex)
+  }
+}
+
+
+export const start = async () => {
+    console.log('Connecting to DB...');
+    const db = await getDB();
+
+    const runNotificationCycle = async () => {
+        const invoices = await db.collection('invoices').find({}).toArray();
+        const tasks = invoices.map((invoice) => new Promise((resolve, reject) => {
+            const { dueDate } = invoice;
+            const today = Date.now();
```

Comment:
Replace 
```const today = Date.now();
            if (today <= dueDate) {```
 with ```
            if (Date.now() <= dueDate) {```
 no need for extra variable if not being re-used anywhere :-)